jerry-core/ecma/builtin-objects/ecma-builtin-global.c

```diff
@@ -804,7 +804,7 @@ ecma_builtin_global_object_decode_uri_helper (ecma_value_t uri __attr_unused___,
 
     if (!lit_read_code_point_from_hex (input_char_p + 1, 2, &decoded_byte))
     {
-      ret_value = ecma_make_throw_obj_completion_value (ecma_new_standard_error (ECMA_ERROR_URI));
+      ret_value = ecma_raise_uri_error ("");
```

Comment:
Same with this type of error. But there is no previous use case in the code base, so I'm not sure what to report.
(Several similar occurences below.)